src/util/query_counters.cpp

```diff
@@ -474,12 +495,20 @@ namespace hpx { namespace util
         }
 
         bool destination_is_cout = false;
+        bool no_output = false;
 
         {
             std::lock_guard<mutex_type> l(mtx_);
             destination_is_cout = destination_ == "cout";
+            no_output = destination_ == "none";
         }
 
+#if HPX_HAVE_ITTNOTIFY != 0 && !defined(HPX_HAVE_APEX)
+        // don't generate any console-output if the ITTNotify API is used
+        if (!no_output && destination_is_cout && use_ittnotify_api)
+            no_output = true;
```

Comment:
is this really what we want? What about if we want to plot performance counters when not running under vtune?